spec/system/candidate_interface/candidate_with_unsuccessful_application_applies_again_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate with unsuccessful application' do
+  include CandidateHelper
+
+  around do |example|
+    date_that_avoids_clocks_changing_by_ten_days = Time.zone.local(2020, 1, 13)
+    Timecop.freeze(date_that_avoids_clocks_changing_by_ten_days) do
+      example.run
+    end
+  end
+
+  scenario 'Can apply again' do
+    given_the_pilot_is_open
+    and_apply_again_feature_flag_is_active
+    and_i_am_signed_in_as_a_candidate
+
+    when_i_have_an_unsuccessful_application
+    and_i_visit_the_application_dashboard
+    and_i_click_on_apply_again
+    and_i_click_on_start_now
+
```

Comment:
I wonder if this should include the submission, perhaps it would've caught the `references_complete?`  issue?

Reply:
Could be done as a follow-up on the same card, but yeah, we should test submission 👍